### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by malikrasheed** [community/community#109428] [GitHub Education]
### Hi everyone,

I have submitted my valid email, which was accepted, and when I post my student id card that was also successfully submitted . after this there are some instructions are shown that had to be solved ,and one of them was [2fa](https://docs.github.com/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)
and when i approve my 2fa and come back all the instructions were wipe out and that is very difficult to find what was the instructions which has to be followed for more ... 
![github issue](https://github.com/community/community/assets/89785894/f48a8c55-c8b6-425e-b76c-c21532581dad)
please help me to solve this issue ...

**Comment by AmirthaRajan** [comment]
I am facing the same issue. Please let us know how to re-upload the document.

**Reply by ArielM457** [reply]
lo arregle ingresando desde mi cel hacer la verificación de 2 pasos, cambiar mi nombre de git a mi nombre completo y sacando foto desde su opcion no cargando la imagen y funcionó

**Comment by Nbenyair** [comment]
same issue

**Comment by hayyrs66** [comment]
Same here...

**Comment by wutingting0829** [comment]
same issue🥲

**Comment by malikrasheed** [comment]
so, all of us are facing same issue since from the last week after apply , and still we dont know how to solve this problem , is there is any way to communicate with the github service center ?

**Comment by ImInTheICU** [comment]
Yup, first it was my image was too big. (figured this out by viewing the F9 request when submitting got error 413)
So I compressed my file to make it smaller and now I'm getting.

"Your application cannot be reviewed until you fix the following:"
     "Your uploaded proof has already been used previously."

**Comment by sandenson** [comment]
Same issue, except that I had to fill in my full name and billing address. I resized the image and it went through. This shouldn't be happening in the first place, though. Fix your shit, GitHub.

**Comment by anluzhangcs** [comment]
same issue,I hope it can be fixed as soon as possible!

**Reply by hayyrs66** [reply]
My solution was to send the photo of the document using my computer's camera, lol.

**Comment by EthicalPentester** [comment]
same

**Comment by hazzemSaid** [comment]
same

**Reply by hazzemSaid** [reply]
i solved my problem (i scan my paper doc from collage and send it ) it is work

**Comment by Asif2714** [comment]
Same here, any assistance would be appreciated!

**Reply by sandenson** [reply]
A submitted a resized version of the image I originally tried to submit. My request was accepted yesterday, so it worked. @Asif2714

**Reply by Asif2714** [reply]
Hey @sandenson thanks for the heads up! I somehow missed your message, and the trick seemed to work for me too (just need to wait for the confirmation now). Hope everyone gets their issues resolved, and we get a full fix from the GitHub team too. Cheers!

**Reply by nihilisticzebra** [reply]
Mine doesn't even give me an option to upload I have to take a screenshot after it process it looks like a camera from 1995 took it and of course they can't read it

**Comment by MichaelYeung2004** [comment]
the same question

**Comment by ab12140** [comment]
Same problem :(

**Comment by Mehdi-B77** [comment]
Have found a Solution guys, set double authentification , complete biling information and then you need to resize or change the format of the image you sent before . hope it work for you and have a good day .

**Reply by dehac58** [reply]
It has also worked for me

**Comment by Jv2350** [comment]
Doing the double authentication the issue been solve if not check previous submissions or  verify file integrity
